src/main/java/org/opensearch/geospatial/geojson/Feature.java

```diff
@@ -8,9 +8,12 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import lombok.Getter;
+
 /**
  * Feature object represents GEOJSON of type Feature.
  */
+@Getter
```

Comment:
you can also use @Value here, it has few perks on top of Getter, https://projectlombok.org/features/Value